src/main/java/org/jenkinsci/plugins/ewm/definitions/DiskPool.java

```diff
@@ -71,6 +84,13 @@ public FormValidation doCheckDiskPoolId(@QueryParameter String value) {
             return validateRequired(value);
         }
 
+        public FormValidation doCheckWorkspaceTemplate(@QueryParameter String value) {
```

Comment:
Good practice: Restrict Stapler methods. Otherwise they become a part of the public UI